suggestions/app_stores.json

```diff
@@ -11,5 +11,19 @@
         "label": "Droid-ify",
         "source": "f",
         "repo": "https://github.com/Iamlooker/Droid-ify"
+    },
+    {
+        "id": "com.aurora.adroid",
+        "label": "Aurora Droid",
+        "reason": "An alternative F-droid client following the same design guidelines as Aurora Store",
+        "source": "f",
+        "repo": "https://gitlab.com/AuroraOSS/auroradroid"
+    },
+    {
+        "id": "com.machiav3lli.fdroid",
+        "label": "Neo Store",
+        "source": "f",
+        "reason": "A modern F-droid client with a material design interface. Forked from Droid-ify",
+        "repo": "https://github.com/NeoApplications/Neo-Store"
```

Comment:
## Differences between NeoStore, Aurora-droid and Droid-ify

| Criterias      | Droid-ify (reference)         | Aurora-droid                | Neo-store      |
|----------------|-------------------------------|-----------------------------|----------------|
|Actively developed                 | ✅            | ❌            | ✅ |
|Polished design                    | ✅            | ✅            | ✅ |
|Material You theming               |✅            | ❌           | ✅ |
|Background installation            |Root, Shizuku |Root, Aurora Services| Root, Shizuku |
|Favorites                          | ✅            | ❌            | ✅ |
|Has multiple default repositories  | ✅            | ✅            | ✅ |
|"More apps by developer" feature   | ❌            | ✅            | ✅ |
| "Update all" feature              | ✅            | ✅            | ✅ |
|Fast startup                       | ✅            | ⚠️            | ⚠️ |
| Search filters                    | Repositories  | ❌            | Repositories, anti-features |

## Per-application special features

| Droid-ify (reference)         | Aurora-droid                | Neo-store      |
|------------------------------|------------------------------|------------------------------|
| Focused on having a simple but beautiful F-droid client | F-droid client following the Aurora Store Design and UX guidelines | F-droid client with a big focus on the User Experience (easier to understand and use than Droid-ify, from my personal experience) |

Reply:
I think that from this comparison between the 3 apps, we can easily exclude Aurora Droid from the database

Reply:
Wrong info, as of now Neo Store [doesn't support shizuku](https://github.com/NeoApplications/Neo-Store/issues/102).
AM support "_Maybe in a future release, but won't come with 1.0.0 (except if someone PRed it)_"

Notable additional features:
 - Disable screenshots
 - Import/Export favourites, ignore updates, _repositories_ (alpha)
 - Custom download folder
 - _Parallel sync of repos_ (alpha)
 - _Parallel downloads from different repos_ (alpha)

Reply:
About "Polished design", I don't think you need any comparison here. Droid-ify made with simple design in mind whereas NeoApplications made for everything customizable.

Reply:
Supporting Shizuku shouldn't be considered a feature because Shizuku, after all, is a non-free app (rather, we should add `reason` if Shizuku is an opt-in feature which isn't the case for Droid-ify).

Last time I checked, Neo Store did not support legacy installers which means it does not support installing through framework-supported alternative installers (eg. default installer app, App Manager).

> AM support "Maybe in a future release, but won't come with 1.0.0 (except if someone PRed it)"

AM will release an SDK soon (with support for unattended updates) which might make it easier for them to support it.

Reply:
> Last time I checked, Neo Store did not support legacy installers...

I'm using their legacy installer feature for a while now.
@DodoLeDev ~Background installation~ replace with "Installer Types" to the table

Reply:
Neo Store, IMO, suffers from various usability issues compared to Droid-ify:

1. UI looks too bloated. I think the authors lack understanding of major and minor features which caused the issue (minor features should receive less focus than the major ones and therefore should be less prominent)
   1. App info (a major feature) is a bottom sheet dialog which should be a new page instead
   2. Distorted/unpolished UI. Examples: coloured buttons wrapped inside a card with a background colour, needless spaces, too much show/hide features some of which should be displayed by default and some via a dialog, large inline buttons.
   3. Some features violate material guidelines (eg. buttons in the dialogs)
2. No lazy loading in the explore page (which takes forever to load)

These usability issues make Neo Store less preferable than Droid-ify.
3. No listing of signatures in case there are multiple providers
4. Latest changes located at the bottom of the app info dialog

Reply:
Same with their Neo Backup (also looks bloated), but we suggest that in our list because no better alternative available.
In this case I vote for Droid-ify.

Reply:
> Wrong info, as of now Neo Store doesn't support shizuku.

Oopsi! I deduced this too quickly because Neo Store is a fork of Droidify 😬

> @DodoLeDev Background installation replace with \"Installer Types\" to the table

✅ Done

Reply:
> Neo Store, IMO, suffers from various usability issues compared to Droid-ify:
> 
> 1. UI looks too bloated. I think the authors lack understanding of major and minor features which caused the issue (minor features should receive less focus than the major ones and therefore should be less prominent)
>    1. App info (a major feature) is a bottom sheet dialog which should be a new page instead
>    2. Distorted/unpolished UI. Examples: coloured buttons wrapped inside a card with a background colour, needless spaces, too much show/hide features some of which should be displayed by default and some via a dialog, large inline buttons.
>    3. Some features violate material guidelines (eg. buttons in the dialogs)
> 2. No lazy loading in the explore page (which takes forever to load)
> 3. No listing of signatures in case there are multiple providers
> 4. Latest changes located at the bottom of the app info dialog
> 
> These usability issues make Neo Store less preferable than Droid-ify.

Hummmm.... Now that you said it, I have to admit that Neo Store is not as good as I thought...

I think I will make the switch to Droidify 😅

Reply:
✅ Droidify wins the F-droid clients battle! I will remove my changes in the next commits